.gitignore

```diff
@@ -2,4 +2,4 @@ tags
 test.sh
 .luarc.json
 nvim
-plugin/packer_compiled.lua
+lazy-lock.json
```

Comment:
Hey! Thanks for the awesome setup, this is awesome for the community! 

I think the lazy-lock should probably be committed to maintain versions across multiple machines?

Reply:
I don't think I want to commit mine. Users can commit their own though by modifying the gitignore.

Reply:
Sounds good!

Reply:
@tjdevries I have the same problem for LazyVim. I don't add the lockfile to .gitignore and simply never add mine to the repo. This way for users, they won't forget to add the lockfile to git.

Reply:
Yeah, I'm just thinking contributors and others will end up adding it on accident constantly haha